addons/stock/models/stock_production_lot.py

```diff
@@ -10,17 +10,17 @@ class ProductionLot(models.Model):
     _description = 'Lot/Serial'
 
     name = fields.Char(
-        'Serial Number', default=lambda self: self.env['ir.sequence'].next_by_code('stock.lot.serial'),
-        required=True, help="Unique Serial Number")
-    ref = fields.Char('Internal Reference', help="Internal reference number in case it differs from the manufacturer's serial number")
+        'Lot/Serial Number', default=lambda self: self.env['ir.sequence'].next_by_code('stock.lot.serial'),
```

Comment:
If I receive a product with a lot L01 (may be a box with 5 ice-creams), now in my local system the lot must be of 3 digit, so I put the number as L001, so in this case we have to keep lot/serial number.
If I am wrong please correct me.